### DISCUSSION TITLE
Generic Placeholder in Rack Elevation

### DISCUSSION THREAD

**Discussion by geaclaesson** [netbox-community/netbox#10508] [General]
Hi there, I've googled and searched the threads here and I couldn't find an answer, so I'm asking my question here.

As I'm building my rack elevations I would like to add Generic placeholders. In my case, it's a UPC that I don't really care about serial and so on, but for the sake of argument, we can say we're talking about blank face plates. 

With a multitude of racks going to be built int netbox, I don't want a database cluttered with duplicates of the same device on new rows with new id's. Am I missing a way of using the same device id in multiple racks?

Thanks for your help!

**Comment by mtinberg** [comment]
It may be clutter but just instantiating a bunch of Device records for different generic sized blanking plates if you wanted to show those rack units as occupied, should work fine, you only need Role, Type, Site and Status to create a Device, and generic-blanking-plate-1u, -2u, etc. could be a type.  Store the data that helps your automation and process, and make sure there is a use-case for "clean" data, otherwise don't bother modeling and entering it.  Did you mean UPS?  If you create a generic 4U UPS Device Type because you don't care whether you have an APC or whatever that's fine and you can drop instances of that device wherever it needs two occupy space, or say generic customer device that you don't manage, but you care about the space used, generic-server-1u, ...  These kinds of devices aren't going to have IPs you track so your automation can only look for devices with has_primary_ip=True and they won't have any negative effect on your config mangement.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: geaclaesson ***@***.***>
Sent: Thursday, September 29, 2022 11:50 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] Generic Placeholder in Rack Elevation (Discussion #10508)


Hi there, I've googled and searched the threads here and I couldn't find an answer, so I'm asking my question here.

As I'm building my rack elevations I would like to add Generic placeholders. In my case, it's a UPC that I don't really care about serial and so on, but for the sake of argument, we can say we're talking about blank face plates.

With a multitude of racks going to be built int netbox, I don't want a database cluttered with duplicates of the same device on new rows with new id's. Am I missing a way of using the same device id in multiple racks?

Thanks for your help!
